extensions/github/src/links.ts

```diff
@@ -91,6 +86,12 @@ function getFileAndPosition(context: LinkContext): IFilePosition | INotebookPosi
 	return undefined;
 }
 
+function getRangeOrSelection(lineNumber: number | undefined) {
+	return lineNumber !== undefined && (!vscode.window.activeTextEditor || vscode.window.activeTextEditor.selection.isEmpty || !vscode.window.activeTextEditor.selection.contains(new vscode.Position(lineNumber - 1, 0)))
```

Comment:
yes, the link code is duplicated between desktop and vscode.dev :/

Reply:
there's been discussion of centralizing it but it would be a lot more work than we're willing to invest (at least as of the last time it was discussed)